### ISSUE TITLE
Elaborate strategy for main vs pulumi-master branches

### ISSUE THREAD

**Issue by t0yv0** [pulumi/terraform#7]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Do we continue development on main?

Specifically do we want to abandon these commits?

```
git log main..pulumi-master | grep commit | wc -l                                                                                                                          ~/code/pulumi-terraform
      13
```

**Comment by t0yv0** [comment]
@Frassle

**Comment by t0yv0** [comment]
Oh interesting so this exposes all the internal/* as pkg already, and it's up to date with v1.4.3. This makes sense. Let me try to use it for my objchange work.

**Comment by Frassle** [comment]
Yup the intention of the "main" branch is "terraform but with pkg/ instead of internal/" I've only made one other change which is renaming "rpcFriendlyDiag" because we also have that type pulled in to tfbridge from somewhere else and `gob.Register` complains about that.

**Comment by t0yv0** [comment]
It's reasonable, I'm taking it for a spin.
